### DISCUSSION TITLE
Is there any data collected by the swift-dependencies library?

### DISCUSSION THREAD

**Discussion by stremsdoerfer** [pointfreeco/swift-dependencies#150] [Q&A] [answered]
Hi !

In order for us to provide accurate Privacy Nutrition Labels to Apple during our review, we need to know whether all our third party libraries collect any kind of data. Is there any data collected by the swift-dependencies library that we should specify in our review? (such as crash, usage, diagnostics etc.)

Thanks!

**Comment by mbrandonw** [comment, marked as answer]
Hi @stremsdoerfer, I've addressed this question in your other [discussion](https://github.com/pointfreeco/swift-identified-collections/discussions/64#discussioncomment-7712996).

**Reply by stremsdoerfer** [reply]
Thank you! Assuming you mean the answer is the same for this library.

**Reply by mbrandonw** [reply]
Yes, same answer.
